docs/source/contributing/version-policy.md

```diff
@@ -72,8 +72,10 @@ Volto runs using [Node.js](https://nodejs.org/en).
 Volto always supports only [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule).
 We recommend using the current LTS version.
 
+- Node.js 22 LTS: Supported since Volto 18.
 - Node.js 20 LTS: Supported since Volto 17.
 - Node.js 18 LTS: Supported since Volto 17.
+- Node.js 18: No longer supported. It was supported in Volto 16 - 17.
```

Comment:
You need to update line 77.

Reply:
I'd remove it, as the new line covers it all.